packages/elements/src/components/drawer-mutant/drawer-mutant.component.ts

```diff
@@ -69,11 +73,11 @@ export class MutationTestReportDrawerMutant extends LitElement {
   private renderDetail() {
     return html`<ul class="list-group">
       ${this.mutant?.killedByTests?.map(
-        (test) => html`<li title="This mutant was killed by this test" class="list-group-item">🎯 ${describeTest(test)}</li>`
+        (test) => html`<li class="list-group-item mx-6" title="This mutant was killed by this test">🎯 ${describeTest(test)}</li>`
       )}
       ${this.mutant?.coveredByTests
         ?.filter((test) => !this.mutant?.killedByTests?.includes(test))
-        .map((test) => html`<li class="list-group-item" title="This mutant was covered by this test">☂️ ${describeTest(test)}</li>`)}
+        .map((test) => html`<li class="list-group-item mx-6" title="This mutant was covered by this test">☂️ ${describeTest(test)}</li>`)}
```

Comment:
Could you remove the bootstrap classes like list-group that are still left over?